packages/salesforcedx-vscode-core/package.json

```diff
@@ -285,6 +289,10 @@
         {
           "command": "sfdx.force.function.start",
           "when": "sfdx:project_opened && resource =~ /.*/functions/.*/[^/]+(/[^/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
+        },
+        {
+          "command": "sfdx.create.manifest",
+          "when": "sfdx:project_opened"
```

Comment:
I'm not seeing the option to create manifest in the edited right clicked menu for files in an sfdx project.  Is that what this option is intended to configure? 
![Screen Shot 2022-01-28 at 10 38 16 AM](https://user-images.githubusercontent.com/76090802/151586265-f7229b8e-5f8f-488f-a25d-85e4ba18d56e.png)

Reply:
I believe that functionality is what's intended here - the right click option to create manifests should only work in the explorer

Reply:
Gotcha.  I'm still burning in the vscode terminology. Is it assumed that if we don't expose a command in the pallet then it also isn't available in the editor context menu?